public/app/features/alerting/unified/components/rule-editor/RuleFolderPicker.tsx

```diff
@@ -16,7 +16,6 @@ export interface Folder {
 
 export interface RuleFolderPickerProps extends Omit<FolderPickerProps, 'initialTitle' | 'initialFolderId'> {
   value?: Folder;
-  dissalowSlashes: boolean;
```

Comment:
not related to this PR, but removed because we are not using it anymore. We are disallowing slashes always.